packages/components/date-picker/src/date-picker-com/panel-date-pick.vue

```diff
@@ -396,7 +396,9 @@ const handleMonthPick = async (month: number) => {
   } else {
     currentView.value = 'date'
     if (['month', 'year', 'date', 'week'].includes(selectionMode.value)) {
-      emit(innerDate.value, true)
+      if (!disabledDate || !disabledDate(innerDate?.value?.toDate())) {
+        emit(innerDate.value, true)
+      }
```

Comment:
Maybe we should cancel this selected operation.

Reply:
At first, I thought so too.

It was later discovered that not all dates of the current month cannot be selected. If users disable dates before November 15th, they will still need to click the November tab to select a different date for that month.

Since the disabled date is determined by the function, if you want to automatically select the first optional date, you need to traverse all the days of the month to achieve.

So in the end I opted to skip the automatic allocation